package.json

```diff
@@ -94,29 +94,28 @@
       "(blocks|components|date|editor|element|i18n|utils)/**/*.js"
     ],
     "coveragePathIgnorePatterns": [
-      "<rootDir>/[^/]+/build/index.js",
-      ".*/story/[^/]+.js"
```

Comment:
I can't see any `/story/` subfolders inside folders which are scanned when generating code coverage.

Reply:
I imagine this is left-over from when we used React Storybook. Thinking the newer `docutron` directory should be similarly ignored.

Reply:
‘docutron’ folder is already ignored with the current setup, thanks for clarifying 👍